src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  width: 100%;
```

Comment:
```suggestion
  font-family: "Roboto", sans-serif;
```
You can specify the font family here to get it inherited

Reply:
still not fixed